polygon-daemon/values.yaml

```diff
@@ -0,0 +1,72 @@
+#values file for the polygon node
+
+replicas: 1
+configMap: scripts-cm
+
+# filesystemId for the storage class
+fileSystemId: ""
+
+chain: mainnet
+
+parts:
+- name: heimdall
+  label: heimdall
+  containerName: heimdall
+  image: 0xpolygon/heimdall:0.3.4
+  pvcMountPath: /heimdall-home
+  pvcMountName: heimdall-storage
+  env:
+  - name: SNAPSHOT_URL
+    value: "" # https://snapshot-download.polygon.technology/heimdall-mainnet-incremental-compiled-files.txt
+  - name: MONIKER
```

Comment:
Is this set for anything beyond aesthetic? I might consider removing to keep the values as small as possible without values that are unlikely to need override.